CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Improvements
 * Added support for Numpy 2 and h5py 3.12, and pinned PyNWB to <3.0 temporarily. [#536](https://github.com/NeurodataWithoutBorders/nwbinspector/pull/536)
+* Added a section for describing the issues with negative timestamps in `TimeSeries` [#545](https://github.com/NeurodataWithoutBorders/nwbinspector/pull/545)
```

Comment:
0.6.1. has been released. Can you move this to the # Upcoming section

Reply:
Oh, no, it was not.

I would suggest to putting dates on the changelog I think it would make it easier to detect this kind of thing:

https://github.com/catalystneuro/neuroconv/blob/main/CHANGELOG.md#v065-november-1-2024

Reply:
Moved it to upcoming.

Reply:
Good point - I can start adding dates.